locals.tf

```diff
@@ -45,6 +45,16 @@ locals {
       protocol  = "udp"
       to_port   = 51835
     },
+    ao_vpn_endpoints_2_tcp = {
+      from_port = 60000
+      protocol  = "tcp"
+      to_port   = 60100
+    },
+    ao_vpn_endpoints_2_udp = {
+      from_port = 60000
+      protocol  = "udp"
+      to_port   = 60100
+    },
```

Comment:
What about simply expanding the current set of port numbers, i.e. something like this?

```hcl
    ao_vpn_endpoints_tcp = {
      from_port = 51820
      protocol  = "tcp"
      to_port   = 51935
    },
    ao_vpn_endpoints_udp = {
      from_port = 51820
      protocol  = "udp"
      to_port   = 51935
    },
```

Reply:
I picked `60000-60100` arbitrarily just to have easier numbers to work with our applications. So `-51935` is possible but less desirable

Reply:
Do you still want to keep the original 51820-51835 UDP range?  If not, then we can go with this instead:

```hcl
    ao_vpn_endpoints_tcp = {
      from_port = 60000
      protocol  = "tcp"
      to_port   = 60100
    },
    ao_vpn_endpoints_udp = {
      from_port = 60000
      protocol  = "udp"
      to_port   = 60100
    },
```

Reply:
Yes, let's keep those - I don't want to break existing functionality at the moment.

Reply:
Sounds good.  Can you please update your change to look like this (preserves alphabetical order for the keys and renames them to align with how we would do it)?

```hcl
    ao_vpn_endpoints_tcp = {
      from_port = 60000
      protocol  = "tcp"
      to_port   = 60100
    },
    ao_vpn_endpoints_udp_1 = {
      from_port = 51820
      protocol  = "udp"
      to_port   = 51835
    },
    ao_vpn_endpoints_udp_2 = {
      from_port = 60000
      protocol  = "udp"
      to_port   = 60100
    },

Reply:
https://github.com/cisagov/cool-sharedservices-openvpn/pull/76/commits/5f6a9b13612303f2fd73e468c6f1a6a19bb450da looks good. 👍